istio/README.md

```diff
@@ -3,17 +3,25 @@
 ## Overview
 
 Datadog monitors every aspect of your Istio environment, so you can:
-- Assess the health of Envoy and the Istio control plane with logs ([see below](#log-collection)).
-- Break down the performance of your service mesh with request, bandwidth, and resource consumption metrics ([see below](#metrics)).
+- Assess the health of Envoy and the Istio control plane with [logs](#log-collection).
+- Break down the performance of your service mesh with [request, bandwidth, and resource consumption metrics](#metrics).
 - Map network communication between containers, pods, and services over the mesh with [Network Performance Monitoring][1].
 - Drill into distributed traces for applications transacting over the mesh with [APM][2].
 
-To learn more about monitoring your Istio environment with Datadog, [see the Istio blog][3].
+To learn more about monitoring your Istio environment with Datadog, [see the Monitor blog post][3].
 
 ## Setup
 
 Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][4] for guidance on applying these instructions.
 
+This OpenMetrics-based integration has a latest mode (`use_openmetrics`: true) and a legacy mode (`use_openmetrics`: false). To get all the most up-to-date features, Datadog recommends enabling the latest mode. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][25].
+
+When you enable the `use_openmetrics` configuration option, the Istio integration uses the latest version of the OpenMetrics-based implementation of the check. 
+
+If you have multiple instances of Datadog collecting Istio metrics, make sure you are using the same version for all of them. Otherwise, metrics data may fluctuate on the Datadog site.
```

Comment:
```suggestion
If you have multiple instances of Datadog collecting Istio metrics, make sure you are using the same mode for all of them. Otherwise, metrics data may fluctuate on the Datadog site.
```